viewer/viewer-awt/src/main/java/org/icepdf/ri/common/SwingViewBuilder.java

```diff
@@ -2362,16 +2355,13 @@ protected JButton makeToolbarButton(
         return tmp;
     }
 
-    private void setPreferredButtonSize(Component comp, String imagesSize) {
-        if (iconSize.equals(Images.SIZE_LARGE)) {
```

Comment:
This is going to need to be fixed up.    Button bar currently only has two sizes so maybe the same logic can apply for now but as is, it breaks layout for small icons.

Reply:
I'm guessing you're referring to me setting `new Dimension(32,32)` regardless of the icon size, right? Yeah, that's a mishap; I'll fix this and implement your other comments in the following days.

Reply:
I believe that buttons should already stretch or shrink according to the size of their icons, so I've removed this method now (only three places depended on it, I believe).